exchanges/binance/binance_wrapper.go

```diff
@@ -604,6 +621,53 @@ func (b *Binance) GetOrderInfo(orderID string) (order.Detail, error) {
 	return orderDetail, common.ErrNotYetImplemented
 }
 
+// GetClosedOrderInfo retrieves specified closed order information
+func (b *Binance) GetClosedOrderInfo(getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
+	if getOrdersRequest.OrderId == "" {
```

Comment:
Can you please add in getOrdersRequest.Validate() check above. [For Reference](https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/order/orders.go#L738) has basic checking. You can also add a specific check ie orderId == "" by adding a method to order.GetOrdersRequest and passing that into the validate function. [Example Here](https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/order/orders.go#L700)

Reply:
Done